policies/gac/gac_networks.py

```diff
@@ -56,7 +56,29 @@ def forward(self, state, taus, actions):
 class Critic(tf.keras.Model):
     def __init__(self, num_inputs, num_networks=1):
```

Comment:
Both for consistency and in order to make debugging and later coding easier, is it alright if you add some comments to the callabalble functions in this class? You can look at the classes created by Liu Yang and I for reference, if you want. Otherwise I think this looks pretty good. I still need to write some unit tests to ensure things work properly.

Reply:
Just did. Does it work?